test/framework/parallelbuild.py

```diff
@@ -63,6 +63,21 @@
 time_cmd = %(time)s
 """
 
+
+MOCKED_SBATCH = """#!/bin/bash
+if [[ $1 == '--version' ]]; then
+    echo "slurm 17.0"
+else
+    echo "Submitted batch job $RANDOM"
+    echo "(submission args: $@)"
+fi
+"""
```

Comment:
It would probably be safer to use bourne-shell syntax and /bin/sh here. The code is simple enough not to need bash.

And same goes for the mocked scontrol

Reply:
Doesn't matter much in the tests? As long as Travis has `/bin/bash`, we're good?

Reply:
Yeah, was just a thought.